modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -44,44 +44,51 @@ spec:
 
 .Procedure
 
-To use the tool on the command line: 
+To use the tool on the command line:
 
-. Run the following command:
+. From the terminal, run:
++
+----
+podman login registry.redhat.io
+----
++
+
+. Run:
 +
-[source,terminal]
 ----
-$ ./cluster-capacity --kubeconfig <path-to-kubeconfig> \ <1>
-    --podspec <path-to-pod-spec> <2>
+podman pull registry.redhat.io/openshift4/ose-cluster-capacity
 ----
-<1> Specify the path to your Kubernetes configuration file.
-<2> Specify the path to the sample `Pod` spec file
 +
-You can also add the `--verbose` option to output a detailed description of how
-many pods can be scheduled on each node in the cluster:
+
+. Run:
 +
 [source,terminal]
 ----
-$ ./cluster-capacity --kubeconfig <path-to-kubeconfig> \
-    --podspec <path-to-pod-spec> --verbose
+$ podman run -v $HOME/.kube:/kube:Z -v $(pwd):/cc:Z  cluster-capacity
```

Comment:
The short-name of the image should match the one we pulled - ose-cluster-capacity 

`podman run -v $HOME/.kube:/kube:Z -v $PWD/cc:/cc:Z ose-cluster-capacity /bin/cluster-capacity --kubeconfig /kube/config --podspec /cc/pod-spec.yaml --verbose`

Reply:
Thanks again, Martin. The fix is in place. @martineg